sensor/src/main/java/org/concord/sensor/ExperimentConfig.java

```diff
@@ -96,6 +96,24 @@
 	 */
 	public float getDataReadPeriod();
 		
+	/**
+	 * This is the number of milliseconds the client should wait between
+	 * starting an experiment and beginning to collect the data in
+	 * addition to the time it takes to collect the first sample.
+	 * 
+	 * @return
+	 */
+	public int getInitialReadDelay();
+		
+	/**
+	 * This is the number of milliseconds the client should wait between
+	 * starting an experiment and beginning to collect the data in
+	 * addition to the time it takes to collect the first sample.
+	 * 
+	 * @return
+	 */
+	public void setInitialReadDelay(int initialReadDelayMillis);
+		
```

Comment:
I added it because at one point it wouldn't build without it, but apparently subsequent changes have obviated the need for it, because now it seems to build without it.